README.md

```diff
@@ -35,4 +32,3 @@ The documentation of the routines is available at <http://permon.vsb.cz/document
 Supported PETSc versions
 ----------------------------------
 PERMON tries to support newest versions of PETSc as soon as possible. The [releases](https://github.com/permon/permon/releases) are tagged with major.minor.sub-minor numbers. The major.minor numbers correspond to the major.minor release numbers of the supported PETSc version.
-
```

Comment:
There should be a new line here.

Reply:
I wonder why but that's not worth further discussion.

Reply:
If I try to edit the file, the newline is there. It seems it's just some GitHub noise. Closing.

Reply:
I think that GitHub shows the new line as missing making the last line not being a POSIX line. Not a big deal in README. It should not happen in a code as some compilers still throw warnings (e.g., CRAY).